### ISSUE TITLE
In LeoSettings.leo, @bool outline-tabs-show-close = True says Ctrl-w closes window but it does not

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3341]
The node `@bool outline-tabs-show-close = True` contains this comment:
```
Show close box (the 'X') on outlien tabs - they can be closed with
Ctrl-W or middle mouse click, so this optionally hides the
redundant 'X'
```

I expected that &lt;Ctrl-w> would close windows because many other programs use the same shortcut for that.  But despite the comment, &lt;Ctrl-w> does not do so (or seem to have any other effect).  I would like it to work that way.

Is it this way to avoid problems with VIM mode?  Or can we go ahead and bind the key to _outline-tabs-show-close_?

**Comment by tbpassin** [comment]
I didn't realize that CTRL-w selects the word surrounding the cursor or selection.  That seems like a sensible and useful binding to me.  So I suggest that the comment in the `@bool outline-tabs-show-close` setting node just be edited to remove the words.  Better might be to have CTRL-w close an outline but only when the focus is in the outline's tab.  Currently after an outline tab has been clicked, CTRL-w acts on the last-focused pane (if it's not the body or tree), or the body if it was the body or tree.

If the focus could actually go to the clicked tab, then a binding of the close command to CTRL-w would make sense.  I don't know if that can be arranged or not.

**Comment by edreamleo** [comment]
Fixed via rev a2b06f01e in devel, including fixing a typo.

This setting is dubious, but I'll leave it alone.
